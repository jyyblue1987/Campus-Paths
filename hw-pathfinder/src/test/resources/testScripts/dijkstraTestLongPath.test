# Find a path with Dijkstra's algorithm when the path is long

CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C
AddNode graph D
AddNode graph E
AddNode graph F
AddNode graph G
AddNode graph H

AddEdge graph A B 1.0
AddEdge graph B C 1.0
AddEdge graph C D 1.0
AddEdge graph D E 1.0
AddEdge graph E F 1.0
AddEdge graph F G 1.0
AddEdge graph G H 1.0

FindPath graph A H