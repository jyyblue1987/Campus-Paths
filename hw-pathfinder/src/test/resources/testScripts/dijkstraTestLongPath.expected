# Find a path with Dijkstra's algorithm when the path is long

created graph graph

added node A to graph
added node B to graph
added node C to graph
added node D to graph
added node E to graph
added node F to graph
added node G to graph
added node H to graph

added edge 1.000 from A to B in graph
added edge 1.000 from B to C in graph
added edge 1.000 from C to D in graph
added edge 1.000 from D to E in graph
added edge 1.000 from E to F in graph
added edge 1.000 from F to G in graph
added edge 1.000 from G to H in graph

path from A to H:
A to B with weight 1.000
B to C with weight 1.000
C to D with weight 1.000
D to E with weight 1.000
E to F with weight 1.000
F to G with weight 1.000
G to H with weight 1.000
total cost: 7.000